Detectors/FIT/FT0/workflow/src/DigitReaderSpec.cxx

```diff
@@ -49,14 +49,17 @@ void DigitReader::run(ProcessingContext& pc)
 {
 
   std::vector<o2::ft0::Digit> digits, *pdigits = &digits;
+  std::vector<o2::ft0::DetTrigInput> trgInput, *ptrTrgInput = &trgInput;
   std::vector<o2::ft0::ChannelData> channels, *pchannels = &channels;
   mTree->SetBranchAddress("FT0DIGITSBC", &pdigits);
   mTree->SetBranchAddress("FT0DIGITSCH", &pchannels);
-
+  if (mUseTrgInput)
+    mTree->SetBranchAddress("TRIGGERINPUT", &ptrTrgInput);
   o2::dataformats::MCTruthContainer<o2::ft0::MCLabel> labels, *plabels = &labels;
   if (mUseMC) {
     mTree->SetBranchAddress("FT0DIGITSMCTR", &plabels);
   }
+
   mTree->GetEntry(0);
```

Comment:
Just noticed that you read only event 0.  There is no particular reason to have this limitation: Note that if the decoded digits are sent to the digitWritier, it will store them as 1 entry per TF.
Could you modify it to work like https://github.com/AliceO2Group/AliceO2/blob/9f9fc26cffc8bb6be5f902ac5fe0fd689b3f6b23/Detectors/FIT/FT0/workflow/src/RecPointReaderSpec.cxx#L46-L56 ?

Reply:
Ok, I only added trigger input to reader, and decided to keep other parts of code untouched. But I can modify it.